pkg/controller/controller_test.go

```diff
@@ -73,7 +78,8 @@ func (s *fakeApiUpdater) applyStorageClass(sc *storagev1.StorageClass) (*storage
 }
 
 func (s *fakeApiUpdater) applyDaemonSet(ds *appsv1.DaemonSet, expectedGeneration int64, forceRollout bool) (*appsv1.DaemonSet, bool, error) {
-	s.daemonSets = append(s.daemonSets, ds)
+	dsRollout := daemonSetRollout{forceRollout: forceRollout, daemonSet: ds}
```

Comment:
I can see `forceRollout` only written, but it does not seem to be read anywhere in the test

Reply:
fixed